src/transxchange/read.rs

```diff
@@ -168,15 +187,100 @@ fn load_companies(transxchange: &Element) -> Result<CollectionWithId<Company>> {
     Ok(companies)
 }
 
+fn load_commercial_physical_modes(
+    transxchange: &Element,
+) -> Result<(CommercialMode, PhysicalMode)> {
+    let mode = match transxchange
+        .try_only_child("Services")?
+        .try_only_child("Service")?
+        .try_only_child("Mode")
+        .map(Element::text)
+    {
+        Ok(mode) => MODES.get(mode.as_str()).unwrap_or(&DEFAULT_MODE),
+        Err(e) => {
+            warn!("{}", e);
```

Comment:
proposal
`warn!("{} - Default mode '{}' assigned", e, DEFAULT_MODE);`

Reply:
I like it better indeed. Done in 50d910bd02903b8a4123c6fd20b9cc5e5044227f.